### PR TITLE
encode URI for json + add a .gitignore for locale devs

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Thomas Labarussias <issif+github@gadz.org>

Encode the URI for drivers with a `+` in their names
Add a .gitignore for locale devs

### CODE REVIEW

File: images/update-drivers-website/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.json
```
Comment (maxgio92) [inline images/update-drivers-website/.gitignore:1]: Good catch!

**Review by maxgio92 — APPROVED** [review]
/approve

**Comment by maxgio92** [comment]
As discussed with @Issif let's anticipate the related cron job for testing

**Comment by Issif** [comment]
As the workload is quick and not big, we thought to increase the frequency up to every 2h, wdyt @leogr @FedeDP ?

File: config/jobs/update-drivers-website/update-drivers-website.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 periodics:
   - name: update-drivers-website
-    cron: '0 12 * * *'  # every day at 12pm
+    cron: '0 */2 * * *'  # every 2h
```
Comment (maxgio92) [inline config/jobs/update-drivers-website/update-drivers-website.yaml:3]: For me is fine. It's a lightweight job. @FedeDP @leogr @zuc what do you think?

File: images/update-drivers-website/.gitignore (hunk 1 above)
Comment (FedeDP) [reply images/update-drivers-website/.gitignore:1]: Mmh do we need this gitignore? Nothing is being pushed to test-infra :/

File: config/jobs/update-drivers-website/update-drivers-website.yaml (hunk 2 above)
Comment (FedeDP) [reply config/jobs/update-drivers-website/update-drivers-website.yaml:3]: It's ok for me!

**Review by maxgio92 — APPROVED** [review]
/approve

**Pull request merged**